### DISCUSSION TITLE
Blue Screen

### DISCUSSION THREAD

**Discussion by janste1978** [rocksdanister/lively#1464] [Q&A] [answered]
I have installed the new update 2.6 from lively wallpaper with .net core 7. After installation the computer cant be use. 10 minutes after start I become a blue screen with hypervisor error or a black screen. If I want to restart the system the computer starts at the third time and the automatic repair. After deinstallation from lively wallpaper and core 7 the computer has no problems.

**Comment by rocksdanister** [comment, marked as answer]
>I have installed the new update 2.6

There is no v2.6 version of Lively, latest update is v2.0.5.0.. from where did you download?

> After installation the computer cant be use. 10 minutes after start I become a blue screen with hypervisor error or a black screen. If I want to restart the system the computer starts at the third time and the automatic repair. After deinstallation from lively wallpaper and core 7 the computer has no problems.

Lively makes no system modification, runs as non-admin user and starts as normal program at the end of system startup; any issue you are experiencing is unlikely due to Lively (driver issue, faulty hardware etc..)

**Comment by janste1978** [comment]
Oh sorry I mean that one, if it was the last update. it was the last update from the program itself. I have activated the autoamtic update function. But I have tryed the version in microsoft store, too. It comes to the same problem. I don't know what is exactly the problem.... the program lively wallpaper or the ,net core 7 installation but one of them or both are the problems in my computer
